core/SymbolRef.h

```diff
@@ -111,7 +111,7 @@ class SymbolRef final {
     // Stores the symbol's Kind and Index. Kind occupies the upper bits.
```

Comment:
Please change this comment to align with the code.

Reply:
Can do.